refapp/SKWebApi/Model/Ask.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace SemanticKernel.Service.Model;
+
+public class Ask
```

Comment:
Should this class and its members be renamed? @craigomatic

Reply:
It would be nice to align with the ContextVariable naming where possible, although I would suggest defer this until after we have the end-to-end solution in the branch.